### DISCUSSION TITLE
Feature Request: C-API implement duckdb_execute_pending_arrow()

### DISCUSSION THREAD

**Discussion by wangfenjin** [duckdb/duckdb#6105] [Ideas]
Why do you want this feature?
Similar to [duckdb_execute_prepared_arrow](https://duckdb.org/docs/api/c/api#duckdb_execute_prepared_arrow), we can query pending execute result by arrow format

**Comment by hannes** [comment]
We would be happy to add this, can you drop me an email at hannes@duckdblabs.com? Thanks!
